test/e2e/framework/util.go

```diff
@@ -2405,10 +2405,17 @@ func WaitForStableCluster(c clientset.Interface, masterNodes sets.String) int {
 
 	}
 	allPods.Items = currentPods
+	systemPods, err := c.CoreV1().Pods(metav1.NamespaceSystem).List(metav1.ListOptions{})
+	ExpectNoError(err)
```

Comment:
please annotate this error with a description.
[1]

Reply:
done